.gitignore

```diff
@@ -2,4 +2,8 @@ __pycache__
 **.pyc
 venv
 src/reference_patents/
+**static
+**uploads
```

Comment:
probably overkill, but as I was moving things I was not sure if these folders had to be move into src/, seems not